index.bs

```diff
@@ -797,6 +797,20 @@ An aggregatable trigger data is a [=struct=] with the following items:
 
 </dl>
 
+<h3 dfn-type=dfn>Aggregatable values configuration</h3>
+
+An aggregatable values configuration is a [=struct=] with the following items:
+
+<dl dfn-for="aggregatable values configuration">
+: <dfn>values</dfn>
+:: An [=ordered map=] whose [=map/key|keys=] are [=strings=] and whose [=map/value|values=] are non-negative 32-bit integers.
```

Comment:
```suggestion
:: A [=map=] whose [=map/key|keys=] are [=strings=] and whose [=map/value|values=] are non-negative 32-bit integers.
```
Since the order is irrelevant.